R/create_monthly_costs.R

```diff
@@ -52,17 +57,24 @@ create_monthly_costs <- function(data,
       values_fill = 0
     ) %>%
     dplyr::select(
-      tidyselect::any_of(
-        month.abb[c(4:12, 1:3)] %>%
-          tolower() %>%
-          paste0("_cost")
-      ),
+      tidyselect::any_of(full_cost_col),
       "daycase_check"
     )
 
   available_months <- setdiff(names(daycase_cost_months), "daycase_check")
+  add_months <- setdiff(full_cost_col, available_months)
+
+  add_months_df <- data.frame(matrix(0, nrow = nrow(data), ncol = length(add_months)))
+  colnames(add_months_df) <- add_months
+
+  daycase_cost_months <- daycase_cost_months %>%
+    cbind(add_months_df) %>%
```

Comment:
Prefer `dplyr::bind_cols` as it's faster and does some other checks on names and types etc.
I think it should work with no issues here.
```suggestion
    dplyr::bind_cols(add_months_df) %>%
```